lib/membrane/rtp/header.ex

```diff
@@ -58,29 +47,20 @@ defmodule Membrane.RTP.Header do
 
   @type t :: %__MODULE__{
           version: version(),
-          padding: padding(),
-          extension_header: extension(),
-          csrc_count: 0..15,
           ssrc: RTP.ssrc_t(),
           marker: marker(),
           payload_type: RTP.payload_type_t(),
           timestamp: timestamp_t(),
           sequence_number: sequence_number_t(),
-          csrcs: [non_neg_integer()],
-          extension_header_data: __MODULE__.Extension.t() | nil
+          csrcs: [RTP.ssrc_t()],
+          extension: __MODULE__.Extension.t() | nil
```

Comment:
Hmm, `extension_header_data` doesn't make sense, because it's not an 'extension header', but a header extension. `extension_data`, on the other hand, is problematic because it has itself the 'data' field, so we'd have `extension_data.data`. Also, the extension bit is usually referred to as 'X bit'. Moreover, you can still do `if header.extension do ...`. So I'd leave this as is.